acceptance-tests.md

```diff
@@ -9,25 +9,32 @@ Reference for NERC OpenStack: [https://nerc-project.github.io/nerc-docs/get-star
 
 1. **Request a new account**
     * _As a new user, I should be able to create an account for myself in the NERC._
```

Comment:
The acceptable use notice acceptance tests comes before this one.  The current expectation is that the users see an acceptable use notice when they sign up for the NERC account through the account portal.  @knikolla shoudl be able to specify on what screen this is suppposed to show up, so it can be verified with a manual web browser check.  This should work regardless of whether someone is requesting an account via their university, Git or other affilliation.

Reply:
@knikolla do you know where the AU notice will be displayed now?  It is important to document this  the acceptance tests.

Reply:
We do not need to document location on the screen, but acceptance test needs to verify that it is visible.  Verify this when we walk through acceptance tests (or note that it is not possible to find).